extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -27,6 +27,7 @@ export const databaseSchemaProvider = 'DatabaseSchemaProvider';
 export const noOpenProjectMessage = localize('noProjectOpenMessage', "No open database project");
 export const projectNodeName = localize('projectNodeName', "Database Project");
 export const dataSourcesNodeName = localize('dataSourcesNodeName', "Data Sources");
+export const referencesNodeName = localize('referencesNodeName', "References");
```

Comment:
since we only have dacpac refs so far not proj ref - should we name it differently?

Reply:
Sure. I changed it to Database References for now and updated the screenshot in the description